packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
-import { addDays, daysFromTodayTo, toMoment, convertDateFormat, getPosition, isMobile } from '@deriv/shared';
+import { addDays, daysFromTodayTo, toMoment, convertDateFormat, getPosition } from '@deriv/shared';
 import Input from './date-picker-input';
 import Calendar from './date-picker-calendar';
 import Native from './date-picker-native';
-import MobileWrapper from '../mobile-wrapper';
-import DesktopWrapper from '../desktop-wrapper';
 import { useOnClickOutside } from '../../hooks/use-onclickoutside';
 import moment, { MomentInput } from 'moment';
 import { TDatePickerOnChangeEvent } from '../types';
+import { useDevice } from '@deriv/hooks';
```

Comment:
@farhan-nurzi-deriv I didn't find `useDevice` hook in `@deriv/hooks`, are you sure it's correct package? I've used `useDevice` hook from `@deriv-com/ui` package

Reply:
Hi, i created a new hook. Is the one from deriv-com/ui has the new breakpoints?

Reply:
https://github.com/binary-com/deriv-app/pull/14346/files#diff-40a2b92f0e15c917c7ad146029359b4c2d2d8f441694eecf1b1602b37a3241af

Reply:
@farhan-nurzi-deriv yes, hook from deriv-com/ui has new breakpoints and better to use it instead of creation a new one

Reply:
Oh nice, i will update my PR. Thanks 🙏

Reply:
Hi @sergei-deriv , i tried using deriv-com/ui and i got error while using it with dtrader. Also, i need a new property to check if tablet in portrait or not. For now, i will keep the new hooks. I will think the way to solve it later because i might need to create a pr to deriv-com/ui library.

Reply:
@farhan-nurzi-deriv ok, but honestly, I don't understand why this package doesn't works for you. Maybe you just forgot to add this package to package.json and re-build the app? But anyway, if you really need something more than just defining device type by width, then ok, but I don't like when we have a couple of source of true in the application. If breakpoint will be changed in the future, we will need to change it in some places, not just in the one.

Reply:
I did, it throws error that need to be fixed in the library. Also if i install it in core, test cases are failing 😵

Reply:
@farhan-nurzi-deriv yeah, because you need to add this library to jest.config as in my [PR](https://github.com/binary-com/deriv-app/pull/13396/files#diff-c4f1d9c9d7b982f26536dbd6bf8c0804c1c0011fba8bc6df6e937363acfffa1f) You can see I've installed deriv-com/ui to the core in my PR ))) But ok, if you promise you will change it later, then I agree ))

Reply:
Thanks @sergei-deriv , i will copy it from your pr. Also, after we added `istabletportrait` to deriv-com/ui i will remove the new hook

Reply:
@farhan-nurzi-deriv for now you don't need to use `deriv-com/ui` library and use `jest.config` from my PR if you still need `isTabletPortrait`, don't worry about this for now. We'll do it later when deriv-com/ui library is updated. For now you can keep it as now